src/frontends/tensorflow_common/src/op/pack.cpp

```diff
@@ -16,12 +20,22 @@ namespace tensorflow {
 namespace op {
 
 OutputVector translate_pack_op(const NodeContext& node) {
-    default_op_checks(node, 1, {"Pack", "PACK"});
+    default_op_checks(node, 1, {"Pack", "PACK"}, true);
     auto num_size = static_cast<int>(node.get_input_size());
 
     auto axis = node.get_attribute<int64_t>("axis", 0);
+    auto complex_type_mark = as_type_ptr<ComplexTypeMark>(node.get_input(0).get_node_shared_ptr());
     auto axis_const = make_shared<v0::Constant>(element::i64, Shape{}, axis);
 
+    if (complex_type_mark) {
+        auto zero = create_same_type_const_scalar<int32_t>(axis_const, 0);
+        auto less_than_zero = make_shared<v1::Less>(axis, zero);
+        auto const_one = make_shared<v0::Constant>(element::i32, Shape{}, 1);
+
+        auto axis_update = make_shared<v1::Select>(less_than_zero, const_one, zero); 
+        auto new_axis = make_shared<v1::Subtract>(axis_const, axis_update);  
+    } 
```

Comment:
we should update `axis` instead of `axis_const` because updated axis will be used during creation of Concat below